### ISSUE TITLE
Clicking on the "Check" button doesn't submit the form

### ISSUE THREAD

**Issue by thbar** [olivierlacan/depend#1]
Quick feedback - I first thought the app wasn't working for some reason, but it's just an issue with the submit button. Great work otherwise, this is useful!

**Comment by olivierlacan** [comment]
Yes! On mobile only right? I noticed this the other day on Mobile Safari and @hone reproduced it on Mobile Firefox I think. It's weird. I thought I forgot to put a form field but it's there. My super terse HTML must be messing with something.

**Comment by olivierlacan** [comment]
Huh, weird. It works on mine... 

(Try > for quoting BTW :-))

**Comment by thbar** [comment]
Hehe :smile: (good point on the quote, I was in a hurry and couldn't remember that :P)
